packages/create-expo-app/src/createAsync.ts

```diff
@@ -78,7 +71,15 @@ async function setupDependenciesAsync(projectRoot: string, props: Pick<Options,
   }
 }
 
-export async function createAsync(inputPath: string, props: Options): Promise<void> {
+export async function createAsync(inputPath: string, options: Options): Promise<void> {
+  if (options.example) {
+    return await createExampleAsync(inputPath, options);
+  }
+
+  return await createTemplateAsync(inputPath, options);
+}
+
+async function createTemplateAsync(inputPath: string, props: Options): Promise<void> {
```

Comment:
Should assert that template and example cannot be used together.

Reply:
Makes sense, will add it tomorrow!